examples/web/nextjs-chat/src/utils/momento-web.ts

```diff
@@ -35,17 +35,20 @@ let onErrorCb: (
   error: TopicSubscribe.Error,
   subscription: TopicSubscribe.Subscription,
 ) => Promise<void>;
-
+let username: string;
 type MomentoClients = {
   topicClient: TopicClient;
 };
 
 async function getNewWebClients(): Promise<MomentoClients> {
   webTopicClient = undefined;
   // we don't want to cache the token, since it will expire in 5 min
-  const fetchResp = await fetch(window.location.origin + "/api/momento/token", {
-    cache: "no-store",
-  });
+  const fetchResp = await fetch(
+    window.location.origin + `/api/momento/token?username=${username}`,
+    {
+      cache: "no-store",
```

Comment:
doesn't this header need to come back in the response?

Reply:
Oh you mean for us to store it? This part is simply sending the `username` while generating the disposable token and gets back the `token` in the `fetchResp`.

Reply:
I'm talking about the `cache` header. That's a response header that browsers look at to determine if it can cache responses or not. It looks like you're providing it in the request, which i don't think will do anything.

Reply:
Oh lol yea that was as it is from before i am not sure at all what that does. there's a comment on this which i think it signals it to not cache?

`  // we don't want to cache the token, since it will expire in 5 min`